src/ui/public/vis/lib/least_common_interval.test.ts

```diff
@@ -26,7 +26,6 @@ describe('leastCommonInterval', () => {
     expect(leastCommonInterval('1000ms', '1s')).toBe('1s');
     expect(leastCommonInterval('1500ms', '1s')).toBe('3s');
     expect(leastCommonInterval('1234ms', '1s')).toBe('617s');
-    expect(leastCommonInterval('1s', '2m')).toBe('2m');
```

Comment:
Why remove this assertion? It's still valid, right?

Reply:
Ah, yes, we learned that seconds are not calendar units from @polyfractal after thinking that they were.